src/libs/Authentication.ts

```diff
@@ -25,7 +25,9 @@ type Parameters = {
 
 let isAuthenticatingWithShortLivedToken = false;
 let isSupportAuthTokenUsed = false;
-Onyx.connect({
+
+// We have used `connectWithoutView` here because it is not connected to any UI
```

Comment:
This comment doesn't provide a whole lot of value. How about something like:

> These session values are only used to help the user authentication with the API. Since they aren't connected to a UI anywhere, it's OK to use connectWithoutView()